app/src/main/java/org/glucosio/android/tools/SplitDateTime.java

```diff
@@ -20,42 +20,44 @@
 
 package org.glucosio.android.tools;
 
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
 import java.text.DateFormat;
-import java.text.SimpleDateFormat;
 import java.util.Date;
 
 public class SplitDateTime {
 
-    private Date origDateTime; // Example "yyyy-MM-dd HH:mm"
+    private DateTime origDateTime;
     private DateFormat inputFormat;
 
-    public SplitDateTime(Date origDatetime, DateFormat origDateFormat) {
-        this.origDateTime = origDatetime;
-        this.inputFormat = origDateFormat;
+    public SplitDateTime(Date origDatetime) {
+        this.origDateTime = new DateTime(origDatetime);
     }
 
     public String getHour() {
-        DateFormat finalFormat = new SimpleDateFormat("HH");
```

Comment:
http://stackoverflow.com/questions/2654025/how-to-get-year-month-day-hours-minutes-seconds-and-milliseconds-of-the-cur